index.yaml

```diff
@@ -165,4 +184,8 @@ entries:
     urls:
     - https://github.com/rstudio/helm/releases/download/rstudio-pm-0.2.0-rc01/rstudio-pm-0.2.0-rc01.tgz
     version: 0.2.0-rc01
+<<<<<<< HEAD
```

Comment:
Something's not quite right here.

Reply:
Sorry about that! Thanks! It failed and I re-ran the action, but by then another commit had fired `cr` and caused a change to index, which created the merge conflict 🙈 I apparently botched the resolution haha